source/connection.c

```diff
@@ -94,13 +94,12 @@ static struct aws_http_connection *s_connection_new(
             } else if (aws_string_eq_byte_buf(s_alpn_protocol_http_2, &protocol)) {
                 version = AWS_HTTP_VERSION_2;
             } else {
-                AWS_LOGF_ERROR(
+                AWS_LOGF_WARN(
                     AWS_LS_HTTP_CONNECTION,
-                    "static: Unrecognized ALPN protocol '" PRInSTR "'.",
+                    "static: Unrecognized ALPN protocol '" PRInSTR "'. Assuming Http/1.1",
                     AWS_BYTE_BUF_PRI(protocol));
```

Comment:
while you're in here, maybe log the string separately at DEBUG level to reduce risk of log-injection attack